tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -80,12 +80,11 @@ variables:
 - name: ReleaseVersionSuffix
   value: ''
 - name: win_trt_version
-  value: 11.8
```

Comment:
should not be changed. i was told that this yml is for producing 11.8 packages. there is another one for 12.x

Reply:
This one is for default cuda package version, which is now cuda 12.2